commit 883aeaa8913855d27d6e6ba50624a30182d8cb44
Author: Eric Anholt <eric@anholt.net>
Date:   Mon Oct 3 10:21:17 2016 -0700

    drivers/vchi: Remove dependency on CONFIG_BROKEN.
    
    The driver builds now.
    
    Signed-off-by: Eric Anholt <eric@anholt.net>

diff --git a/drivers/staging/vc04_services/Kconfig b/drivers/staging/vc04_services/Kconfig
index 9676fb29075a..db8e1beb89f9 100644
--- a/drivers/staging/vc04_services/Kconfig
+++ b/drivers/staging/vc04_services/Kconfig
@@ -1,6 +1,6 @@
 config BCM2708_VCHIQ
 	tristate "Videocore VCHIQ"
-	depends on RASPBERRYPI_FIRMWARE && BROKEN
+	depends on RASPBERRYPI_FIRMWARE
 	default y
 	help
 		Kernel to VideoCore communication interface for the
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1102_883aeaa8913855d27d6e6ba50624a30182d8cb44.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1102_883aeaa8913855d27d6e6ba50624a30182d8cb44.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
